FluentAssertions.Core/Formatting/DateTimeOffsetValueFormatter.cs

```diff
@@ -66,7 +68,34 @@ public string ToString(object value, bool useLineBreaks, IList<object> processed
                 fragments.Add("0001-01-01 00:00:00.000");
             }
 
-            return "<" + string.Join(" ", fragments.ToArray()) + ">";
+            if (HasDate(dateTime))
+            {
+                fragments.Add(FormatOffset(dateTime.Offset));
+            }
+
+            return "<" + string.Join(" ", fragments.Where(f => !string.IsNullOrWhiteSpace(f))) + ">";
+        }
+
+        private string FormatOffset(TimeSpan offset)
+        {
+            if (offset == TimeZoneOffset)
+                return null;
+
+            if (offset == TimeSpan.Zero)
+                return "UTC";
+
+            var sign = offset < TimeSpan.Zero ? "-" : "+";
```

Comment:
Example how parens should be used with conditionals:

`var sign = (offset < TimeSpan.Zero) ? "-" : "+";`